contracts/utils/ContractType.sol

```diff
@@ -15,5 +15,7 @@ enum ContractType {
   /* 10 */ RONIN_TRUSTED_ORGANIZATION,
   /* 11 */ BRIDGE_MANAGER,
   /* 12 */ BRIDGE_SLASH,
-  /* 13 */ BRIDGE_REWARD
+  /* 13 */ BRIDGE_REWARD,
+  /* 14 */ FAST_FINALTIY_TRACKING,
+  /* 15 */ PROFILE
```

Comment:
I cannot see the profile contract anywhere

Reply:
Already deployed in #266